pkg/ui/frontend/src/reducers/hpCreate.js

```diff
@@ -0,0 +1,246 @@
+                ...state,
+                parameterConfig: params,
+            }
+        case actions.EDIT_PARAMETER_HP:
+            params = state.parameterConfig.slice();
+            params[action.index][action.field] = action.value;
+            return {
+                ...state,
+                parameterConfig: params,
+            }
+        case actions.DELETE_PARAMETER_HP:
+            params = state.parameterConfig.slice();
+            params.splice(action.index, 1);
+            return {
+                ...state,
+                parameterConfig: params,
+            }
+        case actions.ADD_LIST_PARAMETER_HP:
+            params = state.parameterConfig.slice();
+            params[action.paramIndex].list.push({
+                value: "",
+            })
+            return {
+                ...state,
+                parameterConfig: params
+            }
+        case actions.EDIT_LIST_PARAMETER_HP:
+            params = state.parameterConfig.slice();
+            params[action.paramIndex].list[action.index] = action.value;
+            return {
+                ...state,
+                parameterConfig: params
+            }
+        case actions.DELETE_LIST_PARAMETER_HP:
+            params = state.parameterConfig.slice();
+            params[action.paramIndex].list.splice(action.index, 1);
+            return {
+                ...state,
+                parameterConfig: params
+            }
+        case actions.CHANGE_WORKER:
+            return {
+                ...state,
+                worker: action.worker,
+            }
+        case actions.CHANGE_ALGORITHM:
+            return {
+                ...state, 
+                suggestionAlgorithm: action.algorithm,
+            }
+        case actions.ADD_SUGGESTION_PARAMETER:
+            let suggestionParameters = state.suggestionParameters.slice();
+            let parameter = {name: "", parameterType: ""};
+            suggestionParameters.push(parameter);
+            return {
+                ...state,
+                suggestionParameters: suggestionParameters,
+            }
+        case actions.CHANGE_SUGGESTION_PARAMETER:
+            suggestionParameters = state.suggestionParameters.slice();
+            suggestionParameters[action.index][action.field] = action.value;
+            return {
+                ...state,
+                suggestionParameters: suggestionParameters,
+            }
+        case actions.DELETE_SUGGESTION_PARAMETER:
+            suggestionParameters = state.suggestionParameters.slice();
+            suggestionParameters.splice(action.index, 1);
+            return {
+                ...state,
+                suggestionParameters: suggestionParameters,
+            }
+        case actions.CHANGE_REQUEST_NUMBER:
+            return {
+                ...state,
+                requestNumber: action.number,
+            }
+        case actions.ADD_METRICS_HP:
+            let metricsName = state.metricsName.slice()
```

Comment:
let's stay consistent and use semicolons

Reply:
Gonna add them everywhere.